ios/docs/Controls/Navigation.md

```diff
@@ -0,0 +1,42 @@
+# Navigation
+
+## Overview
+
+FluentUI introduces the `NavigationBar` and `NavigationController` classes as subclasses of their UIKit counterparts, `UINavigationBar` and `UINavigationController` respectively. These classes will handle rendering all the relevant information with a Fluent look and feel. This means that for the most part, you can tell your app to use the Fluent `NavigationController` (which already knows to use the Fluent `NavigationBar` by default) and things should work for the most part.
```

Comment:
Last sentence feels a bit too casual.

Reply:
Can we also like a general description of what NavigationBar/UINavigationBar are

Reply:
> Last sentence feels a bit too casual.

You say that like it's a bad thing.

Reply:
It is. "for the most part" isn't helpful to read when you're looking at documentation. Neither is "should work".

Reply:
You don't need to redefine too deeply but a one or two sentence description would be nice. It would also align better with the overview section of our other controls